A3.md

```diff
@@ -96,10 +97,10 @@ void grpc_channel_tracer_add_trace(grpc_channel_tracer* tracer,
 
 /* Adds a subchannel to the tracing object */
 void grpc_channel_tracer_add_subchannel(grpc_channel_tracer* tracer, 
```

Comment:
If we decide to track subchannels only via the references in the individual trace nodes, then this method may not be needed.